<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
    <property name="styleSheet">
        <string notr="true">background-color: #FFFFFF;</string>
    </property>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #4CAF50; color: white;</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>login_icon.png</normaloff>
     </iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="signup_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Signup</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2196F3; color: white;</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>signup_icon.png</normaloff>
     </iconset>
    </property>
   </widget>
   <widget class="QWidget" name="login_info_widget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>500</width>
      <height>150</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="username_label">
       <property name="text">
        <string>Username/Phone</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="username_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="password_label">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password_lineEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
